Application/EdFi.Security.DataAccess/Models/ResourceClaimActionAuthorizationStrategies.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text;
+
+namespace EdFi.Security.DataAccess.Models
+{
+    public class ResourceClaimActionAuthorizationStrategies
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int ResourceClaimActionAuthorizationStrategyId { get; set; }
+
+        public int ResourceClaimActionAuthorizationId { get; set; }
```

Comment:
Again, I _think_ this property is superfluous when we already have the reference property below.

Reply:
I updated to ResourceClaimActionId  . this property required here to have foreign key relationship with ResourceClaimAction table